scripts/release/rpm/mariner.dockerfile

```diff
@@ -10,13 +10,12 @@ WORKDIR /azure-cli
 
 COPY . .
 
-# Mariner only has 'python3' package. It has no version-specific packages, like 'python39'.
-# - On Mariner 1.0, 'python3' is Python 3.7
-# - On Mariner 2.0, 'python3' is Python 3.9
+# Mariner 2.0 only has 'python3' package, which is currently (2022-12-09) Python 3.9.
+# It has no version-specific packages like 'python39'.
 RUN dos2unix ./scripts/release/rpm/azure-cli.spec && \
     REPO_PATH=$(pwd) CLI_VERSION=$cli_version PYTHON_PACKAGE=python3 PYTHON_CMD=python3 \
     rpmbuild -v -bb --clean scripts/release/rpm/azure-cli.spec && \
-    cp /usr/src/mariner/RPMS/x86_64/azure-cli-${cli_version}-1.x86_64.rpm /azure-cli-dev.rpm
+    cp /usr/src/mariner/RPMS/x86_64/azure-cli-${cli_version}-1.cm2.x86_64.rpm /azure-cli-dev.rpm
```

Comment:
Why not use fuzzy matching instead?
For example:
`cp /usr/src/mariner/RPMS/x86_64/azure-cli-${cli_version}-1.*x86_64.rpm /azure-cli-dev.rpm`

Reply:
Good question! Actually, we use fuzzy matching for Red Hat:

https://github.com/Azure/azure-cli/blob/bb28eea93ff29033e32ba03aef4783784a737cfa/scripts/release/rpm/fedora.dockerfile#L17

As I described in the PR description, even `.x86_64` may not be standard as other package names contains `-x64`, I am not sure about the boundary/how fuzzy we should use. That's why I still use exact matching.